scripts/flashbots/02-goerli-send-tx.ts

```diff
@@ -0,0 +1,134 @@
+import { Contract, ContractFactory, utils, BigNumber, Signer } from 'ethers';
+import { run, ethers, network } from 'hardhat';
+import {
+  FlashbotsBundleProvider,
+  FlashbotsBundleResolution,
+  FlashbotsTransaction,
+  FlashbotsTransactionResponse,
+  SimulationResponse,
+} from '@flashbots/ethers-provider-bundle';
+import * as contracts from '../../utils/contracts';
+
+const { Confirm } = require('enquirer');
+const prompt = new Confirm('Do you wish to stealth-mint though flashbots on goerli?');
+
+async function main() {
+  await run('compile');
+  await prompt.run().then(async (answer: any) => {
+    await mainExecute();
+  });
+}
+
+function mainExecute(): Promise<void | Error> {
+  return new Promise(async (resolve, reject) => {
+    const [owner] = await ethers.getSigners();
+    console.log('using address:', owner.address);
+
+    const provider = ethers.providers.getDefaultProvider('goerli');
```

Comment:
Be careful, this actually doesnt use the provider URL you give to it. You should use `ethers.provider` afaik, and execute the script with `--network goerli`.

Reply:
yeah, nice catch, I'll add a check that ensures networkId is goerli's